threeML/plugins/OGIP/eventlist.py

```diff
@@ -1062,12 +1113,10 @@ def __init__(self, arrival_times, energies, n_channels, start_time=None, stop_ti
 
             self._dead_time = None
 
-
-
-    def _exposure_over_interval(self, tmin, tmax):
+    def exposure_over_interval(self, tmin, tmax):
```

Comment:
Is this method supposed to be called by the user?

If yes, then you should add a detailed docstring.

If not, then you should call it _exposure_over_interval (also in the parent class).